tools/calyx-pass/src/cli.rs

```diff
@@ -0,0 +1,49 @@
+use argh::FromArgs;
+
+#[derive(FromArgs)]
+#[argh(
+    description = "A tool for visualizing pass transformations.\n\nAuthor: Ethan Uppal"
+)]
+pub struct ParseArgs {
+    #[argh(
```

Comment:
You can also use `///` comments to provide the description instead of using the description field.